content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,62 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTPd" %}}
+
+## HTTPd with Datadog module
+Datadog provides a [module][1] for enhancing [Apache HTTP Server][2] capabilities with APM Tracing.
+
+### Module installation
+<div class="alert alert-warning">
+  <strong>Note:</strong> Only Apache HTTP Server 2.4.x is supported.
+</div>
+
+The module is provided as a shared library for dynamical loading by HTTPd. Each supported platform
+and architecture has its own artifact hosted on [httpd-datadog's repository][1].
+
+Run the following script to download the latest version of the module:
+
+```bash
+get_latest_release() {
```

Comment:
[Minor, non-blocking] Not sure if there are other reasons for doing it, but as a user I would probably prefer one of the one-liners — for example from [here](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8) — rather than defining a function + an env + a command. In case there mis a specific reason, than it is more than fine.

Reply:
I like the one-liner approach better. Thanks for the suggestion.